newrelic/core/config.py

```diff
@@ -1145,29 +1132,34 @@ def error_matches_rules(
     if not settings:
         return False
 
-    # TODO Remove default None after settings are implemented
-    classes_rules = getattr(settings.error_collector, "%s_classes" % rules_prefix, set())
-    status_codes_rules = getattr(
-        settings.error_collector, "%s_status_codes" % rules_prefix, set()
-    )
+    # Retrieve settings based on prefix
+    classes_rules = getattr(settings.error_collector, "%s_classes" % rules_prefix)
```

Comment:
Do we still want a default here to avoid any potential AttributeErrors?

Reply:
Yeah that's fair, I'll put that back in